internal/pkg/externalplugins/autoresponder/autoresponder_test.go

```diff
@@ -212,7 +628,8 @@ func TestHelpProvider(t *testing.T) {
 			configInfoIncludes: []string{":"},
 		},
 	}
-	for _, c := range cases {
+	for _, testcase := range cases {
+		c := testcase
```

Comment:
Maybe it would be better to uniformly abbreviate 'testcase' to 'tc' or 'c'.

Reply:
Fixed.